cmd/pg/daemon.go

```diff
@@ -0,0 +1,70 @@
+package pg
+
+import (
+	"net"
+	"os"
+	"path"
+
+	"github.com/wal-g/wal-g/utility"
+
+	"github.com/spf13/cobra"
+	"github.com/wal-g/tracelog"
+	"github.com/wal-g/wal-g/internal"
+	"github.com/wal-g/wal-g/internal/asm"
+	"github.com/wal-g/wal-g/internal/databases/postgres"
+)
+
+const DaemonShortDescription = "Uploads a WAL file to storage"
+const DaemonSocketName = "/tmp/wal-push.sock"
+
+// daemonCmd represents the daemon command
+var daemonCmd = &cobra.Command{
+	Use:   "daemon wal_filepath",
```

Comment:
What is the logic behind the `wal_filepath` argument? I guess that the WAL-G daemon command should accept the path to the socket as an argument.